### PR TITLE
Adding Behaviour Terms October 8th

### PR DESCRIPTION
**Pull request by matentzn**
This PR contains a large number of changes affecting the behavioral phenotype branch. This work was performed by @bcoleman96, for details contact him.

### CODE REVIEW

File: src/ontology/hp-edit.owl [hunk 1]
```diff
@@ -26952,23 +27086,33 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0000710> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0000711> (Restlessness)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:28332736") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0000711> "A state of unease characterized by diffuse motor activity or motion, subject to limited control, nonproductive, or disorganized behavior.")
```
Comment (matentzn) [inline src/ontology/hp-edit.owl:26955]: Is the loss of PMID acceptable?

File: src/ontology/hp-edit.owl [hunk 2]
```diff
@@ -27029,9 +27186,14 @@ AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0000717> "Th
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0000717> "Autism")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0000717> <http://purl.obolibrary.org/obo/HP_0000729>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_0000718> (Aggressive behavior)
+# Class: <http://purl.obolibrary.org/obo/HP_0000718> (Aggressive behaviour)
```
Comment (matentzn) [inline src/ontology/hp-edit.owl:27189]: UK spelling?

File: src/ontology/hp-edit.owl [hunk 3]
```diff
@@ -87607,16 +88110,22 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0008757> "Unil
 EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0008757> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000763> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0003706>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000634>))))
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0008757> <http://purl.obolibrary.org/obo/HP_0001605>)
 
-# Class: <http://purl.obolibrary.org/obo/HP_0008760> (Violent behavior)
+# Class: <http://purl.obolibrary.org/obo/HP_0008760> (Violent behaviour)
```
Comment (matentzn) [inline src/ontology/hp-edit.owl:88113]: UK spelling?

**Review by matentzn — COMMENTED** [review]
@bcoleman96

File: src/ontology/hp-edit.owl (hunk 1 above)
Comment (bcoleman96) [reply src/ontology/hp-edit.owl:26955]: The PMID wasn't copied to the original spreadsheet where people were editing, but I don't think it is a good reference for this term. So yes, I would leave it out.

File: src/ontology/hp-edit.owl (hunk 2 above)
Comment (bcoleman96) [reply src/ontology/hp-edit.owl:27189]: This should be "behavior". It looks like the synonyms include both spellings.

File: src/ontology/hp-edit.owl (hunk 3 above)
Comment (bcoleman96) [reply src/ontology/hp-edit.owl:88113]: This should be "behavior". It looks like the synonyms include both spellings.

**Pull request merged**